### PR TITLE
Task 6 Notes

### PR DESCRIPTION
**Pull request by Laranto-spb**
I renamed Transaction Model to **MoneyTransaction**, because I got an error with system name "transaction" - needs to reset old db to run.
Features:
1. Create note with new/update transaction
2. Edit, delete note - show transaction
3. List of transactons with icons - with/without note
4. Note is optional for transaction
5. Note has validation

### CODE REVIEW

File: app/helpers/money_transactions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
```
Comment (e-popolamov) [inline app/helpers/money_transactions_helper.rb]: this empty file needed?

File: app/helpers/notes_helper.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
```
Comment (e-popolamov) [inline app/helpers/notes_helper.rb]: and this?

File: spec/requests/notes_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
```
Comment (e-popolamov) [inline spec/requests/notes_spec.rb:4]: why not added test for crud?

Comment (e-popolamov) [reply spec/requests/notes_spec.rb:4]: model crud it is not the same that controller crud :)

**Review by e-popolamov — CHANGES_REQUESTED** [review]

File: app/helpers/notes_helper.rb (hunk 2 above)
Comment (Laranto-spb) [reply app/helpers/notes_helper.rb]: deleted all, except Application. They all were empty

**Comment by e-popolamov** [comment]
also change name transaction_spec and add  more cases

**Review by e-popolamov — APPROVED** [review]

**Pull request merged**